docs/Troubleshooting.md

```diff
@@ -0,0 +1,56 @@
+# Troubleshooting
+
+This page aimed to help you fix the common problems encountered while using Spaceship prompt.
+
+## What's the weird symbol for `git` branch?
+
+You need to have a powerline patched font inorder to properly display `git` branch symbol.
+
+* Install any powerline compatible font like [Fira Code](https://github.com/tonsky/FiraCode) or [others](https://github.com/powerline/fonts).
+* Configure your terminal emulator to [use that font](https://powerline.readthedocs.io/en/master/troubleshooting/osx.html).
+
+## What's the weird characters infront of sections?
```

Comment:
"character in front of a section" sounds better

Reply:
Fixed.